lib/protos.js

```diff
@@ -334,8 +333,16 @@ exports._wrapInitialize = function() {
  */
 
 exports._wrapPage = function() {
-  // Noop the first page call if integration assumes pageview
-  if (this._assumesPageview) return this.page = after(2, this.page);
+  var page = this.page;
+  this.page = function() {
+    if (this._assumesPageview) {
+      if (this._initialPageSkipped) return page.apply(this, arguments);
+
+      return this._initialPageSkipped = true;
```

Comment:
similar to my question #123

what do we expect this method to return ?  It's a bit confusing reading that line, because it appears that we're returning a boolean, but that doesn't appear to be the intention. If the method return null, it might be clearer if we just used two lines, one to set the var, the other to explicitly return.

Reply:
fixed!